### PR TITLE
Capstone Project 3

### PR DESCRIPTION
**Pull request by AxelSoler**
## Monster Hunter World

- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the API is stored in the Redux store.
- Fetch data from [Monster Hunter World API](https://docs.mhw-db.com/)
- Stars and id are numeric values
- [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))
- When the page loads, the webapp shows the list of monsters that could be filtered out by the name.
- When the user clicks on a monster item, the application navigates to the details page.
- When the user clicks on the "Back" button (<), the user navigates to the home page.
- Create unit tests
- [video](https://www.loom.com/share/809cbea31e43403fa25c375040c95ccd)

All feedback is well received

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,66 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to the team
+- All partners from the morning session and standup who gave us some advises
+
```
Comment (ShahierNashaat) [inline README.md]: - [x] Kindly be noticed that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md [hunk 2]
```diff
@@ -1,46 +1,66 @@
...
+- Technologies used: GIT, Github, Gitflow, linters, Jest
+- [Monster Hunter World API](https://docs.mhw-db.com/)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Live Demo
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+[Live Demo Link](https://radiant-taffy-25c53a.netlify.app/)
```
Comment (ShahierNashaat) [inline README.md]: - [x] Kindly be noticed that the project is deployed properly, the home and detail page works as expected, and your link only displays hello.

File: README.md [hunk 3]
```diff
@@ -1,46 +1,66 @@
...
 
-### `npm test`
+## About video
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+[video](https://www.loom.com/share/398a28a1abd1413e81476d184d1cb843)
```
Comment (ShahierNashaat) [inline README.md]: - [x] Kindly be noticed that the video contains a highlight about a piece of the code the student finds interesting or is proud of making.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @AxelSoler , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — APPROVED** [review]
Hi @AxelSoler, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**